notebooks/example.ipynb

```diff
@@ -200,6 +199,143 @@
     "%%bash\n",
     "cat ./fixtures/rendered_config.yaml"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea5eff92",
+   "metadata": {},
+   "source": [
+    "## Example 3: Comparing two config files\n",
+    "Let's explore using the at using the `config.compare()` method to compare two two config files"
```

Comment:
```suggestion
    "Let's explore using the `config.compare()` method to compare two config files"
```

Reply:
Suggestion resolved in above conversation

Reply:
Paul's suggestion had missed the 2 twos, and it looks like that still needs to be updated.

Reply:
Yes it is! Removed one 'two'.